lib/ignore/ignore.go

```diff
@@ -517,7 +533,33 @@ func parseIgnoreFile(fs fs.Filesystem, fd io.Reader, currentFile string, cd Chan
 			continue
 		}
 
-		line = filepath.ToSlash(line)
+		trimmed := strings.ReplaceAll(strings.ReplaceAll(line, " ", ""), "\t", "")
+		if strings.HasPrefix(trimmed, escapePrefix) {
```

Comment:
Do we really want to match `# es cap    e = \` here? It feels like it should be enough to just trim spaces at the ends?